hack/verify-yamllint.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+# Copyright 2020 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
```

Comment:
It is better to add `set -euo pipefail` in the script.

Reply:
`yamllint -f parsable deploy/*.yaml | grep -v "line too long" > /tmp/yamllint.log` will return error(1)

Reply:
btw, cannot fix following issue since some repo path is too long:
```
deploy/crd-csi-driver-registry.yaml:22:81: [error] line too long (88 > 80 characters) (line-length)
```